### DISCUSSION TITLE
Does ekuiper function support generate timestamp or convert format_time_string to timestamp

### DISCUSSION THREAD

**Discussion by silenceli** [lf-edge/ekuiper#2358] [Q&A]
I have searched doc `https://ekuiper.org/docs/zh/latest/sqls/functions/datetime_functions.html`。I couldn't find any built-in functions in EKuiper for generating timestamps or converting time strings to timestamps. I would like to inquire if such functions exist, and if not, how should I proceed?

**Comment by ngjaying** [comment]
https://ekuiper.org/docs/en/latest/sqls/functions/other_functions.html#tstamp
